dashboard/final-example/next.config.js

```diff
@@ -1,4 +1,9 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+
+const nextConfig = {
+  experimental: {
+    ppr: 'incremental',
+  },
+};
```

Comment:
Should this be `.mjs` or is the addressed (and will conflict) with the other PR?

Reply:
Updated with the merge from main 👍🏼